openedx/core/djangoapps/site_configuration/helpers.py

```diff
@@ -29,6 +29,17 @@ def get_current_site_configuration():
         return None
 
 
+def get_current_site_configuration_values(default=None):
```

Comment:
@shimulch, we have to write tests for this and possibly the change to `get_base_template_context`, if it already has one.

Reply:
@lgp171188, I've added test for ``get_current_site_configuration_values``. I don't find any explicit number of key checking in the email context-related tests. Hopefully, we are good here.